projects.go

```diff
@@ -38,57 +38,67 @@ type ProjectsService struct {
+	ImportError                               string              `json:"import_error"`
+	Permissions                               *Permissions        `json:"permissions"`
+	MarkedForDeletionAt                       *ISOTime            `json:"marked_for_deletion_at"`
+	Archived                                  bool                `json:"archived"`
+	AvatarURL                                 string              `json:"avatar_url"`
+	SharedRunnersEnabled                      bool                `json:"shared_runners_enabled"`
+	ForksCount                                int                 `json:"forks_count"`
+	StarCount                                 int                 `json:"star_count"`
+	RunnersToken                              string              `json:"runners_token"`
+	PublicBuilds                              bool                `json:"public_builds"`
+	OnlyAllowMergeIfPipelineSucceeds          bool                `json:"only_allow_merge_if_pipeline_succeeds"`
+	OnlyAllowMergeIfAllDiscussionsAreResolved bool                `json:"only_allow_merge_if_all_discussions_are_resolved"`
+	RemoveSourceBranchAfterMerge              bool                `json:"remove_source_branch_after_merge"`
+	LFSEnabled                                bool                `json:"lfs_enabled"`
+	RequestAccessEnabled                      bool                `json:"request_access_enabled"`
+	MergeMethod                               MergeMethodValue    `json:"merge_method"`
+	ForkedFromProject                         *ForkParent         `json:"forked_from_project"`
+	Mirror                                    bool                `json:"mirror"`
+	MirrorUserID                              int                 `json:"mirror_user_id"`
+	MirrorTriggerBuilds                       bool                `json:"mirror_trigger_builds"`
+	OnlyMirrorProtectedBranches               bool                `json:"only_mirror_protected_branches"`
+	MirrorOverwritesDivergedBranches          bool                `json:"mirror_overwrites_diverged_branches"`
+	ServiceDeskEnabled                        bool                `json:"service_desk_enabled"`
+	ServiceDeskAddress                        string              `json:"service_desk_address"`
+	IssuesAccessLevel                         *AccessControlValue `json:"issues_access_level"`
+	RepositoryAccessLevel                     *AccessControlValue `json:"repository_access_level"`
+	MergeRequestsAccessLevel                  *AccessControlValue `json:"merge_requests_access_level"`
+	ForkingAccessLevel                        *AccessControlValue `json:"forking_access_level"`
+	WikiAccessLevel                           *AccessControlValue `json:"wiki_access_level"`
+	BuildsAccessLevel                         *AccessControlValue `json:"builds_access_level"`
+	SnippetsAccessLevel                       *AccessControlValue `json:"snippets_access_level"`
+	PagesAccessLevel                          *AccessControlValue `json:"pages_access_level"`
```

Comment:
I don't see all these access level fields in the docs, are you sure they exist? Can you point me to a location that confirms that?

Reply:
https://gitlab.com/api/v4/projects/19184008 / https://gitlab.com/gitlab_ap_test/root_project

![image](https://user-images.githubusercontent.com/30216166/88854802-23bb7500-d1f2-11ea-960f-1c06c33010b9.png)

request access to project if you will. (switched to public)

Reply:
Check, thanks for pointing me to this. So then these should be good, but it shouldn't be pointers. If you can update that, we should be good I guess.

Reply:
all `*AccessControlValue` changed to `AccessControlValue` :+1: